jujugui/static/gui/src/app/assets/css/overrides/_overrides.scss

```diff
@@ -0,0 +1,3 @@
+@import './button-row';
+@import './dropdown-menu';
+@import './panel';
```

Comment:
Some the components need GUI specific overrides or fixes for using the .v1 namespace.